src/aap_eda/services/project/git.py

```diff
@@ -210,3 +212,15 @@ def __call__(
             if e.stderr:
                 usr_msg += f" Error: {e.stderr}"
             raise GitError(usr_msg) from None
+
+    def _set_http_proxy(self) -> None:
```

Comment:
@Alex-Izquierdo If these env vars are set in the vm's systemd service file wouldn't they automatically flow to the child process. If we are running these as containers where would the user set these env vars.

Reply:
Hi @mkanoor 
In the case of vm's there are multiple ways to set the envar. In case of containers I'm not sure %100, @rooftopcellist can confirm, but I think the operator allows to add arbitrary environment variables.